hydra-kafka/pom.xml

```diff
@@ -0,0 +1,67 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<!--
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.addthis.hydra</groupId>
+    <artifactId>hydra-parent</artifactId>
+    <version>5.2.5-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>hydra-kafka</artifactId>
+  <name>Hydra Kafka Module</name>
+  <description>reading and writing for kafka 8 clusters</description>
+
+  <dependencies>
+    <!-- addthis deps -->
+    <dependency>
+      <groupId>com.addthis</groupId>
+      <artifactId>basis</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.addthis</groupId>
+      <artifactId>bundle</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.addthis.hydra</groupId>
+      <artifactId>hydra-main-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.addthis.hydra</groupId>
+      <artifactId>hydra-task</artifactId>
+    </dependency>
+
+
+      <!-- 3rd party -->
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka-clients</artifactId>
+      <version>0.8.2.0</version>
```

Comment:
should probably put this version in dependency management or at least a property (since it is repeated and apparently likely to change a few times)